### PR TITLE
refactor: install from downloads.newrelic.com instead of GitHub

### PR DESCRIPTION
**Pull request by ctrombley**

### CODE REVIEW

File: scripts/install.ps1 [hunk 1]
```diff
@@ -0,0 +1,3 @@
+$version = (New-Object System.Net.WebClient).DownloadFile("https://download.newrelic.com/install/newrelic-cli/currentVersion.txt").Trim();
```
Comment (Julien4218) [inline scripts/install.ps1:1]: @ctrombley we need to include this for the older windows versions
[Net.ServicePointManager]::SecurityProtocol = 'tls12, tls';

**Review by Julien4218 — CHANGES_REQUESTED** [review]

File: scripts/install.ps1 (hunk 1 above)
Comment (ctrombley) [reply scripts/install.ps1]: 👍

**Review by sanderblue — DISMISSED** [review]
🐑 it

File: internal/cli/version.go [hunk 2]
```diff
@@ -23,8 +23,9 @@ var (
 	latestVersion string
 )
 
-const installCLISnippetLinux = `curl -Ls https://raw.githubusercontent.com/newrelic/newrelic-cli/master/scripts/install.sh | bash && sudo newrelic install`
-const installCLISnippetWindows = `[Net.ServicePointManager]::SecurityProtocol = 'tls12, tls'; (New-Object System.Net.WebClient).DownloadFile("https://github.com/newrelic/newrelic-cli/releases/latest/download/NewRelicCLIInstaller.msi", "$env:TEMP\NewRelicCLIInstaller.msi"); msiexec.exe /qn /i $env:TEMP\NewRelicCLIInstaller.msi | Out-Null; & 'C:\Program Files\New Relic\New Relic CLI\newrelic.exe' install`
+const installCLISnippetLinux = `curl -Ls https://download.newrelic.com/install/newrelic-cli/scripts/install.sh | bash && sudo newrelic install`
+
+var installCLISnippetWindows = fmt.Sprintf(`[Net.ServicePointManager]::SecurityProtocol = 'tls12, tls'; (New-Object System.Net.WebClient).DownloadFile("https://download.newrelic.com/install/newrelic-cli/v%s/NewRelicCLIInstaller.msi", "$env:TEMP\NewRelicCLIInstaller.msi"); msiexec.exe /qn /i $env:TEMP\NewRelicCLIInstaller.msi | Out-Null; & 'C:\Program Files\New Relic\New Relic CLI\newrelic.exe' install`, version)
```
Comment (Julien4218) [inline internal/cli/version.go:28]: @ctrombley shouldn't we download the new install.ps1 here?

Comment (ctrombley) [reply internal/cli/version.go:28]: Good catch, thanks

**Review by Julien4218 — APPROVED** [review]

**Comment by Julien4218** [comment]
This looks good @ctrombley

**Pull request merged**